samples/tests/test_client_query_job_optional.py

```diff
@@ -14,13 +14,13 @@
 
 import typing
 
-from .. import client_query_shortmode
+from .. import client_query_job_optional
 
 if typing.TYPE_CHECKING:
     import pytest
 
 
 def test_client_query_shortmode(capsys: "pytest.CaptureFixture[str]") -> None:
-    client_query_shortmode.client_query_shortmode()
+    client_query_job_optional.client_query_job_optional()
```

Comment:
Should we add some verification that the query was indeed run in jobless mode?

Reply:
Control over execution is still the responsible of the service, and it can choose to fallback to using jobs regardless of the requested mode.